book/website/collaboration/ppie-events.md

```diff
@@ -31,26 +31,29 @@ It covers:
 
 (cl-ppie-events-background)=
 ## Motivation and Background
-This chapter is meant to serve as a resource to support and guide researchers, organisers, and community members with the tools to identify the challenges and opportunities in organising events that involve and include members of the public, and in particular people with lived experience.
+
+  * This chapter is meant to serve as a resource to support and guide researchers, organisers, and community members with the tools to identify the opportunities and mitigate barriers in the organisation of events that involve and include members of the public, and in particular people with lived experience.
 
 We refer to "people with lived experience" throughout this chapter as a way to recognize and acknowledge individuals who have direct personal knowledge, or firsthand experience of a particular situation that is relevant to the topic of a research-related event. 
-For example, this could relate to a particular medical condition, disability, or other social conditions. 
+For example, this could relate to a particular medical condition, disability, or other social condition. 
 It is a term often used to emphasise the importance of including the perspectives and insights of those who have had direct experience. {cite:ps}nihr2021differentexperiences,camden2022preferredlanguage
 
-This chapter is focused in particular on how to make research events more inclusive and accessible to members of the public and people with lived experience who do not have experience or a background as a researcher, as they can face particular barriers to participation.
+This chapter is focused in particular on how to make research events more inclusive and accessible to members of the public and people with lived experience who do not have experience or a background as a researcher, as they can face particular barriers to participation. 
 
-Many research projects are increasingly involving members of the public in the research. 
```

Comment:
I think this should stay as "involving" to reflect the active involvement practices rather than just engagement/inclusion. There's more of this in general discussion about "Arnstein's Ladder of Citizen Participation"